libs/core/src/components/pds-button/pds-button.scss

```diff
@@ -1,4 +1,6 @@
 :host {
+  --border-radius-default: 9999px;
```

Comment:
We have a semantic token for this: `--pine-border-radius-circle`.

Reply:
I went with `-pine-border-radius-round` since the one you mentioned is in `semantic.scss` and we haven't pulled them in yet.